argo/prow-config.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: prow-config
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/RedbackThomson/ack-test-infra.git
```

Comment:
oops :)

Reply:
Not quite oops. Just for faster iteration. Very much on purpose, will submit a fix up before merge

Reply:
I'd be good to merge this as-is if the above pointed to the main repo. :)